scriptworker/cot/verify.py

```diff
@@ -36,14 +39,16 @@
 
 DECISION_MACH_COMMANDS = ((
     './mach', 'taskgraph', 'decision'
-), (
+), )
+ACTION_MACH_COMMANDS = ((
```

Comment:
@imbstack - should I remove all `ACTION_MACH_COMMANDS` except for `action-callback`? I'm assuming we only want to support cot verification if the action is in `actions.json`, but I want to doublecheck.

Reply:
Definitely true in the long-term. We should hold off on removing them for a bit because of older trees though?